### PR TITLE
[14.0][FIX] payroll_account: Fix multicompanies and refactoring

### PR DESCRIPTION
**Pull request by FernandoRomera**

### CODE REVIEW

**Comment by FernandoRomera** [comment]
Solved #116

**Comment by FernandoRomera** [comment]
@nimarosa 
Can you review?

File: payroll_account/models/hr_contract.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+
+from odoo import fields, models
+
+logger = logging.getLogger(__name__)
```
Comment (mtelahun) [inline payroll_account/models/hr_contract.py]: You don't need a logger if you are not using it.

File: payroll_account/models/hr_payslip_line.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+
+from odoo import models
+
+logger = logging.getLogger(__name__)
```
Comment (mtelahun) [inline payroll_account/models/hr_payslip_line.py]: Same here. No need for logging.

File: payroll_account/models/hr_payslip_run.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+
+from odoo import fields, models
+
+logger = logging.getLogger(__name__)
```
Comment (mtelahun) [inline payroll_account/models/hr_payslip_run.py]: see above

File: payroll_account/models/hr_salary_rule.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+
+from odoo import fields, models
+
+logger = logging.getLogger(__name__)
```
Comment (mtelahun) [inline payroll_account/models/hr_salary_rule.py]: See above

**Review by mtelahun — CHANGES_REQUESTED** [review]
Code review seems Ok, but it's hard to tell because you mixed code changes with refactoring. Mixing code changes with formatting changes makes it hard for reviewers to understand exactly what you changed. Even if you don't want to create separate PRs for them you should at least put them in separate commits (not in one big commit like this one).

**Comment by FernandoRomera** [comment]
> Code review seems Ok, but it's hard to tell because you mixed code changes with refactoring. Mixing code changes with formatting changes makes it hard for reviewers to understand exactly what you changed. Even if you don't want to create separate PRs for them you should at least put them in separate commits (not in one big commit like this one).

Ok, thanks for the explanation.

**Comment by FernandoRomera** [comment]
Replace in models/hr_salary_rule.py
domain=[("deprecated", "=", False)]
by
domain=[("deprecated", "=", False), ("company_id", "=", company_id)],

**Comment by nimarosa** [comment]
@FernandoRomera Like @mtelahun said, we need you to respect OCA guidelines for PRs. I edited the title of the PR, this should also be the commit message, because module names are used for migration scripts and bots. 

Also the refactoring and the changes should be in separated commit but we can pass on that for now. 

Please if you can, do a reword of the commit message to match PR title, and we can merge this.

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst

**Comment by FernandoRomera** [comment]
@nimarosa 
Done, thanks.

**Review by AntoniRomera — APPROVED** [review]

**Review by MRomeera — APPROVED** [review]

**Comment by FernandoRomera** [comment]
@nimarosa 
Can you review? and if is fine merge, thanks.

**Review by nimarosa — APPROVED** [review]

**Pull request merged**